### DISCUSSION TITLE
网络框架

### DISCUSSION THREAD

**Discussion by LogicCoder** [kbengine/kbengine#1346] [Q&A]
之前说打算更新一个帧同步框架，不知道现在还有没有计划，网络这一块不知道能否出一个官方的帧同步或者状态同步框架，这样可以补全maba类或多人对战类游戏的需求

**Comment by kbengine** [comment]
目前没有完成

**Reply by worldg** [reply]
我写一个

**Reply by LogicCoder** [reply]
赞，期待，为了效率这块应该是用C++写吧，实在不熟悉C++

**Reply by qq952670771** [reply]
现在做的游戏，差不多就是你们做的状态同步，更新实在是太频繁了，10s，700伤害和同步，服务器性能要求非常高，意味着成本也要求非常高

**Comment by qhf566** [comment]
谁有大厂,近5年内开发完的,国内没有运营过的,成品开发完毕的手机游戏,联系我809996677
